integration_tests/sdk/data/hotel_reviews.csv

```diff
@@ -0,0 +1,6 @@
+hotel_name,review_date,reviewer_nationality,review
```

Comment:
Are these tables supposed to be so small?

Reply:
I will try to put a larger table and see how it goes against test.